packages/calcite-components/src/components/tabs/resources.ts

```diff
@@ -1,3 +1,7 @@
+export const CSS = {
```

Comment:
As per https://github.com/Esri/calcite-design-system/pull/11720/files#r2004373229 , this is not required.